### PR TITLE
Replace apostrophe with quote in gulpfile.babel.js and webpack.config.js.

### PR DESCRIPTION
**Pull request by fzzzy**
Also, remove an unused import (fs) and remove one dangling comma.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,42 +1,41 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
-const fs = require('fs');
```
Comment (fzzzy) [inline webpack.config.js:3]: Lint told me to remove this fs.

File: webpack.config.js [hunk 2]
```diff
@@ -82,74 +81,74 @@ if (RUN_ANALYZER) {
...
+    allowedHosts: ["example.com"],
+    publicPath: "/",
+    contentBase: path.join(__dirname, "frontend/build"),
+    index: "index.html",
     before: app => {
-      app.use(morgan('dev'));
+      app.use(morgan("dev"));
       app.use(DevServerMiddleware);
     },
-    https: { ...DevServerHTTPSOptions },
```
Comment (fzzzy) [inline webpack.config.js:117]: Lint told me to remove this dangling comma.

File: webpack.config.js (hunk 1 above)
Comment (lmorchard) [reply webpack.config.js:3]: Huh, I'm surprised it's not actually in use, but I can't find where so :+1:

**Review by lmorchard — APPROVED** [review]
LGTM

**Pull request merged**